helm_deploy/values-prod.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# Per environment values which override defaults in hmpps-incentives-api/values.yaml
+
+generic-service:
+  replicaCount: 4
```

Comment:
same as the default value anyway so not necessary

Reply:
👍